filebeat/values.yaml

```diff
@@ -1,38 +1,121 @@
 ---
-# Allows you to add any config files in /usr/share/filebeat
```

Comment:
Can you re-add all these removed key in a deprecated section with empty value like in the metricbeat chart:

https://github.com/elastic/helm-charts/blob/58bdd96312a08cf3e933ffa9afb3c7988c27dd80/metricbeat/values.yaml#L275-L289